cmd/libs/go2idl/types/types.go

```diff
@@ -84,6 +84,14 @@ type Package struct {
 	// package name).
 	Types map[string]*Type
 
+	// Functions within this package, indexed by their name (*not* including
+	// package name).
+	Functions map[string]*Type
```

Comment:
We need to track functions separately because a method can have the same name as a type.